### ISSUE TITLE
building in linux64

### ISSUE THREAD

**Issue by timredfern** [bakercp-archive/ofxOAuth#14]
Hi I tried to build one of the examples from ofxOAuth and I got this error:

<code>
In file included from ../../../addons/ofxOAuth/libs/libcurl/include/curl/curl.h:35:0,
                 from ../../../addons/ofxOAuth/src/ofxOAuth.cpp:37:
../../../addons/ofxOAuth/libs/libcurl/include/curl/curlrules.h: At global scope:
../../../addons/ofxOAuth/libs/libcurl/include/curl/curlrules.h:143:41: error: size of array ‘**curl_rule_01**’ is negative
     [CurlchkszEQ(long, CURL_SIZEOF_LONG)];
                                         ^
</code>

I remove libcurl from the libs folder to use the system version, this worked. I think the included libcurl is a 32 bit version?

**Comment by danomatika** [comment]
Same problem with OS X 64 bit.

**Comment by bakercp** [comment]
This lib is essentially deprecated.  Oauth1 and 2 are supported now in ofxHTTP.

**Comment by danomatika** [comment]
Ok, can you mark it as such? I'm looking into making ofxTwitter work for a student and seems like I have to essentially rebuild it now. Wee.

**Comment by bakercp** [comment]
I have an unpublished twitter addon that uses ofxhttp and oauth1.  give me a few minutes and it may be ready for you ...

**Comment by danomatika** [comment]
That would be great as it appears the current ofxTwitter addons keep getting abandoned :P

**Comment by danomatika** [comment]
Also, if this addon is now legacy, I'd close all the issues and PR's and *loudly* proclaim "this is done, look elsewhere". :)

**Comment by bakercp** [comment]
Yes ... it's somewhere about 87 on my todo list ... as soon as I publish ofxTwitterTools, I'll do it :)
